modules/examples/src/main/java/com/ibm/cloud/platform_services/iam_identity/v1/IamIdentityExamples.java

```diff
@@ -832,5 +838,50 @@ public static void main(String[] args) throws Exception {
             logger.error(String.format("Service returned status code %s: %s\nError details: %s",
                     e.getStatusCode(), e.getMessage(), e.getDebuggingInfo()), e);
         }
+
+        try {
+            System.out.println("createReport() result:");
+
+            // begin-createReport
+
+            CreateReportOptions createReportOptions = new CreateReportOptions.Builder()
+                    .accountId(accountId)
```

Comment:
Please make sure that the examples are consistent across the various languages. For example, @michaelbeck coded the Go example for `CreateReport()` such that it does not set the accountId, but it sets the type and duration parameters.
When comparing the examples across the languages, we want them to be identical other than language-specific differences.